recipes/default.rb

```diff
@@ -276,14 +275,14 @@
       hosts_connect_to: hosts_connect_to,
       mode: avahi_zeroconf_enabled ? 'switch' : network_mode
     )
-    notifies :reload, 'service[tinc]', :delayed
-    notifies :reload, 'ohai[reload_network]', :delayed
+    notifies :reload, 'service[tinc]', :immediately
```

Comment:
I'm not sure actually about this one ... changing the mode or the port of tinc doesn't impact the Ohai network plugin.